### PR TITLE
refactor: remove antd and antd icons

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

- Removed `antd` and `@ant-design/icons` and their remaining usages, to save about 8MB on the final exe size and be able to upload to CF again
- Fixes a potential leak with update state callbacks, clean up update state keeping

### CODE REVIEW

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]
Font size issues in several places like settings, simbridge settings, simbridge release notes etc. Generally font sizes are observed to be smaller, buttons also affected. Please compare with latest release.

|PR| 3.3.8|
|--|--|
|Screenshot (3448)|Screenshot (3450)|
|Screenshot (3455)|Screenshot (3449)|
|Screenshot (3456)|Screenshot (3457)|

Also the full screen crash report that currently occurs on initial start when updating from the latest release to master has UI issues.

|3.3.8 -> PR| 3.3.8 -> master|
|--|--|
|Screenshot (3453)|Screenshot (3459)|

Changes on InstallerUpdate LGTM and are tested by auto-updating PR -> 3.3.8. No issues observed.

**Comment by Benjozork** [comment]
@FoxtrotSierra6829 I fixed the styling issues, however I could not reproduce the issue coming from latest release

File: src/renderer/index.tsx [hunk 1]
```diff
@@ -72,6 +72,8 @@ ipcRenderer.invoke(channels.sentry.requestSessionID).then((sessionID) => {
 
 // Obtain configuration and use it
 InstallerConfiguration.obtain().then((config: Configuration) => {
+    throw new Error('a'.repeat(255));
```
Comment (FoxtrotSierra6829) [inline src/renderer/index.tsx:75]: should be removed

Comment (Benjozork) [reply src/renderer/index.tsx:75]: lol, yeah, probably...

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]
Still some small issues

- _Don't show again_ (tickbox) too small in SimBridge autostart modal (can be disregarded if issue is opened to replace it with the normal _don't show again_ (toggle) version from other modals)
Screenshot (3490)

- _Location for temporary folders_ path to small (visibible if option above is enabled)
Screenshot (3492)

- _Configuration download URL_ path too small

- Error code in modal spills over
Screenshot (3493)

**Review by frankkopp — APPROVED** [review]

**Pull request merged**